### PR TITLE
Use tab-size css property when calculating node size

### PR DESCRIPTION
**Pull request by tmilloff**
First PR please be gentle

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-textarea-autosize",
   "description": "textarea component for React which grows with content",
-  "version": "5.2.0",
+  "version": "5.2.1",
```
Comment (Andarist) [inline package.json]: updating `dist` files and the version is part of the publishing process, should not be done in PRs

**Comment by Andarist** [comment]
Besides the minor comment, I guess this is good to go.

**Comment by tmilloff** [comment]
Thank you! Apologies for running/commiting `dist`; if I submit a PR in the future I will refrain from doing so.

**Comment by Andarist** [comment]
Could u revert last 2 commits and push the branch out?

**Comment by Andarist** [comment]
Thanks for the PR!

**Pull request merged**